catalogValidationValues.json.template

```diff
@@ -4,4 +4,7 @@
   "resource_prefix": $PREFIX,
   "resource_group_name": "validation",
   "region": "us-south",
+  "watson_discovery_plan": "plus",
+  "watsonx_assistant_plan": "plus",
+  "watson_machine_learning_plan": "v2-standard",
```

Comment:
another question, ML is provisioned by default https://github.com/terraform-ibm-modules/terraform-ibm-watsonx-saas-da/blob/8fb100ee51b41b1ed2ede50ab6451a1d2beedfd9/variables.tf#L57

Why do we need to specify this plan? Maybe you want to update `watsonx_governance_plan`.

Reply:
It doesn't hurt to specify the input even if the default happens to be the value we want. Just being explicit for validation. The governance will be added to validation later but it's left out on purpose so it doesn't appear in the estimates for the RAG stack.